http/data-url.json

```diff
@@ -5,22 +5,19 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Basics_of_HTTP/Data_URIs",
         "description": "data URL scheme",
         "support": {
-          "webview_android": {
-            "version_added": true
-          },
           "chrome": {
             "version_added": true
           },
           "chrome_android": {
             "version_added": true
           },
           "edge": {
-            "version_added": "12",
-            "notes": "The maximum size supported is 4GB"
+            "notes": "The maximum size supported is 4GB",
+            "version_added": "12"
```

Comment:
I prefer keeping `version_added` and `version_removed` at the top.

Reply:
I think my editor might have assumed I wanted to sort the grandchildren as well (which is not accurate).  Whoops!